app/assets/stylesheets/application.scss

```diff
@@ -1,18 +1,18 @@
 @import "materialize/components/color";
 
 /*** Additional variables ***/
-$primary-bg-color: #FFF;
-$primary-font-color: #333;
+$primary-bg-color: #F7FCBD;
+$primary-font-color: #3A3335;
 
 /*** Color ***/
-$primary-color: #FFF;
-$secondary-color: color("blue-grey", "darken-1") !default;
+$primary-color: #EFF4AB;
+$secondary-color: #456293 !default;
 $success-color: color("green", "base") !default;
-$error-color: color("red", "base") !default;
-$link-color: color("light-blue", "darken-1") !default;
+$error-color: color("red", "darken-1") !default;
+$link-color: #539fac !default;
 
 /*** Body ***/
-body { background-color: #FAFAFA; }
+body { background-color: #e6e6e6; }
```

Comment:
Color literals like `#e6e6e6` should only be used in variable declarations; they should be referred to via variable everywhere else.